pandas/core/array_algos/masked_reductions.py

```diff
@@ -100,3 +100,9 @@ def min(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
 
 def max(values: np.ndarray, mask: np.ndarray, skipna: bool = True):
     return _minmax(np.max, values=values, mask=mask, skipna=skipna)
+
+
+def mean(values: np.ndarray, mask: np.ndarray, skipna: bool = True, min_count: int = 0):
+    return _sumprod(
+        np.sum, values=values, mask=mask, skipna=skipna, min_count=min_count
+    ) / np.count_nonzero(~mask)
```

Comment:
Can you separate this on multiple lines? (eg calculate sum and count separately, and then divide both on its own line)